Dockerfile

```diff
@@ -1,14 +1,19 @@
 FROM node:16-bullseye
 WORKDIR /engine/
+
+COPY package.json /engine/package.json
+COPY yarn.lock /engine/yarn.lock
+RUN ["yarn", "install"]
+
+RUN ["apt", "update"]
+RUN ["apt", "upgrade", "-y"]
+RUN ["apt", "install", "postgresql-client-13", "-y"]
```

Comment:
I am curious why you went to the array-based syntax instead of the CLI-like syntax.

Reply:
+1

Reply:
I changed the entrypoint in another PR to use the array style (and I remember having problems with the normal style). So I also changed these to match that :D